pkg/controller/template/template_controller.go

```diff
@@ -103,6 +112,65 @@ func New(
 	return ctrl
 }
 
+func (ctrl *Controller) addSecret(obj interface{}) {
+	s := obj.(*v1.Secret)
```

Comment:
could we make this var just called "secret" or something more descriptive instead of `s`